src/services/objections.api.service.ts

```diff
@@ -1,52 +1,46 @@
-import logger from "../utils/logger";
-
```

Comment:
This definitely seems more like a client if it calls the sdk, so we could say
client -> sdk proxy service -> api

Reply:
I followed the approach taken by Appeals
https://github.com/companieshouse/lfp-appeals-frontend/blob/master/src/modules/appeals-service/AppealsService.ts
There they use services which the controllers interact with and those services can then call the modules they need to complete the function.
The idea is to have an objections service that the controllers can interact with.
It also provides loose coupling with the sdk, a single point of contact. If all interactions are with the service then it minimises change if the sdk interface changes - we wouldn't have to update several controllers, only the service.

Reply:
From what you have said this structure seems good - still a bit uneasy about the naming i.e just calling it "service" something more descriptive to reflect what you have said and to distinguish it from the sdk service would be helpful.